internal/cli/uninstall.go

```diff
@@ -41,15 +41,30 @@ func RemoveInstalled(packages []string) {
 		sharedir := utils.ShareDir() // Directory of file
 		pkg := packages[i]           // Current package
 		pkgrcloc := path.Join(sharedir, fmt.Sprintf("%s.yaml", pkg))
+		ctprcloc := path.Join(sharedir, fmt.Sprintf("%s/.catppuccin.yaml", pkg))
```

Comment:
Both of these should probably use `filepath.Join` to account for OS differences.